src/libs/ajax.js

```diff
@@ -443,6 +443,16 @@ const Workspaces = signal => ({
         return fetchRawls(`${root}/checkBucketReadAccess`, _.merge(authOpts(), { signal }))
       },
 
+      checkBucketAccess: async (bucket, accessLevel) => {
+        if (!Utils.canWrite(accessLevel)) {
+          return false
+        }
```

Comment:
Since this call happens inside the launch modal, and the user can only open that if they have compute (write) access, this check is no longer necessary.

Reply:
I'd like to keep this guard in place (along with a comment explaining why) to prevent generating a pet service account token in the project for non-writers of the workspace. Sam is unable to do the protection itself because it only asks for the project, not the workspace.